src/rollup/index.ts

```diff
@@ -163,284 +162,290 @@ function getInputOptions(rawInputOptions: GenericConfigObject): any {
 	return inputOptions;
 }
 
+let curWatcher: Watcher;
+export function setWatcher(watcher: Watcher) {
+	curWatcher = watcher;
+}
```

Comment:
This indirection is done so that setting the watcher is a private API and not exposed.

Alternativel the main rollup function could just be wrapped.

Reply:
I thought about this but yes, this seems like a reasonable approach.

Reply:
On a totally unrelated note, I wonder why the diff in Github looks so messy. IntelliJ is able to create a nice and focused diff for this file. I hope Github can improve this at some point...

Reply:
Just discovered that if you select "Diff settings -> Hide Whitespace Changes -> Reload" this gets cleaned up nicely!